apps/projects/templates/a4_candy_projects/project_detail.html

```diff
@@ -21,6 +21,17 @@
 {% endif %}
 {% endblock %}
 
+<!-- ensure cookie overlay for embedded videos -->
+{% block extra_css %}
+  {{ block.super }}
+    <link rel="stylesheet" href="{% static 'dsgvo_video_embed.css' %}" />
```

Comment:
Can you remove it from `livequestion_module_detail.html` again then? Otherwise we load it twice.

Reply:
oh good point, done.

Reply:
Thanks!